### PR TITLE
Fix value of points in Question #399

### PR DESCRIPTION
**Pull request by draganakovacevic**
There was option to enter decimal numbers but now it's fix to only accept integer numbers. Range of numbers cannot be negative because it was previously set with @Min and @Max annotations.

### CODE REVIEW

**Comment by stanislavgrujic** [comment]
@aberonja @BraneOpacic21 Please review.

File: web/src/main/frontend/src/scss/layout/_panel.scss [hunk 1]
```diff
@@ -213,12 +218,16 @@ $border: 1px solid white;
...
+  z-index: 1;
+  position: fixed;
+  bottom: 180px;
   width: 100%;
+  height: 75px;
   button.icon__block {
     margin-left: -15px;
+    position: inherit;
+    height: initial;
+    z-index: inherit;
```
Comment (BraneOpacic21) [inline web/src/main/frontend/src/scss/layout/_panel.scss:230]: Please, double-check if these 3 lines are needed at all

File: web/src/main/frontend/src/scss/components/_formula.scss [hunk 2]
```diff
@@ -48,12 +51,13 @@
...
 
   .formula__categories_block {
-    max-height: 240px;
+    position: inherit;
     margin-top: 20px;
-    overflow-y: scroll;
     margin-bottom: 20px;
     width: 100%;
-    z-index: 10;
+    z-index: 15;
```
Comment (BraneOpacic21) [inline web/src/main/frontend/src/scss/components/_formula.scss:58]: As we discussed prior, lets increment the `z-index` by one each time, so we don't end up (as our app grows) with unnecessary large numbers which are hard to track

File: web/src/main/frontend/src/index.js [hunk 3]
```diff
@@ -24,8 +24,10 @@ import {inviteEmployeeForm} from './components/forms/InviteEmployeeForm';
 
 class Root extends React.Component {
   handleCopyEvent(e) {
-    e.clipboardData.setData('text/plain', 'No cheating! :)');
-    e.preventDefault();
+    if(GuestStore.testId != null && GuestStore.testId != '') {
+      e.clipboardData.setData('text/plain', 'No cheating! :)');
+      e.preventDefault();
+    }
```
Comment (BraneOpacic21) [inline web/src/main/frontend/src/index.js:30]: I suppose that we would want to _prevent the default behavior_ of the event regardless of the `if statement`, right? 🤔  Correct me if I'm wrong.

Also, the condition could probably be written a bit simpler:

`if(GuestStore.testId) {
// do something
}`

or

`if(!GuestStore.testId) {
// do something
}`

depending on the implementation you choose

File: web/src/main/frontend/src/scss/layout/_panel.scss [hunk 4]
```diff
@@ -213,12 +218,16 @@ $border: 1px solid white;
 }
 
 .panel__add_button {
-  z-index: 8;
-  position: absolute;
-  bottom: 60px;
+  z-index: 1;
```
Comment (BraneOpacic21) [inline web/src/main/frontend/src/scss/layout/_panel.scss:221]: Double-check please, if `z-index` is not necessary here, remove it

**Pull request merged**